Jellyfin.Server/Program.cs

```diff
@@ -683,27 +688,15 @@ private static string GetUnixSocketPath(IConfiguration startupConfig, IApplicati
             return socketPath;
         }
 
+        [UnsupportedOSPlatform("windows")]
         private static void SetUnixSocketPermissions(IConfiguration startupConfig, string socketPath)
         {
             var socketPerms = startupConfig.GetUnixSocketPermissions();
 
             if (!string.IsNullOrEmpty(socketPerms))
             {
-                #pragma warning disable SA1300 // Entrypoint is case sensitive.
-                [DllImport("libc")]
-                static extern int chmod(string pathname, int mode);
-                #pragma warning restore SA1300
-
-                var exitCode = chmod(socketPath, Convert.ToInt32(socketPerms, 8));
-
-                if (exitCode < 0)
-                {
-                    _logger.LogError("Failed to set Kestrel unix socket permissions to {SocketPerms}, return code: {ExitCode}", socketPerms, exitCode);
-                }
-                else
-                {
-                    _logger.LogInformation("Kestrel unix socket permissions set to {SocketPerms}", socketPerms);
-                }
+                File.SetUnixFileMode(socketPath, (UnixFileMode)Convert.ToInt32(socketPerms, 8));
```

Comment:
Should this be wrapped in try-catch?

Reply:
https://github.com/jellyfin/jellyfin/pull/8547/files/f03f15c94ca23ce0b6d14bf7ed9b8fef6b58bb18#diff-3f5208ac08350a0b582312c67d6c021442c16dfdb43460397033f8fc75e56d5dR211
This would catch it and stop the server